docs/FAQ.md

```diff
@@ -32,11 +32,15 @@ as well. Hopefully contributed by the community.
 
 And note that implementations of Backstage can be hosted wherever you feel suits your needs best.
 
-## Can Backstage by used for other things than developer portals?
+## Q: Can Backstage by used for other things than developer portals?
 
 Yes. 
 
 The core frontend framework could be used for building any large-scale web application where multiple teams are building separate parts of the app, but you want the overall experience to be consistent.
 
 That being said, in [Phase 2](https://github.com/spotify/backstage#project-roadmap) of the project we will add features that are needed for developer portals and systems for managing software ecosystems. Our ambition will be to keep Backstage modular.
 
+## Q: My company doesn't have thousands of developers. Is Backstage overkill?
+
+Sure, having something like Backstage gets more important as the number of developers in your company grows. One of the core reasons to adopt Backstage is to help standardise how software is built at your company. Setting guidelines and deciding on standards is easier when your company is smaller.
```

Comment:
Can be read as "Sure it's overkill".

Reply:
+1 How about `"Absolutely. Having..."`

Reply:
As a rule of 👍 , I'd always phrase the question in a way that we want the answer to be yes.

So in this case "My company doesn't have thousands of developers, is Backstage still useful to us?"

Reply:
It's probably fine for now, but I feel the main argument here should be that Backstage comes with a lot of integrations out of the box so it's easier to adopt Backstage as a package rather than gluing together many tools for your developers by yourself. Of course, this hinges on us actually delivering on the promise of having many well-integrated plugins for industry-standard tooling out of the box.

Reply:
Updated.